app/src/main/java/com/github/onlynotesswent/ui/common/Popups.kt

```diff
@@ -248,81 +299,41 @@ fun FileSystemPopup(
-                                  }
-                                  .padding(12.dp)) {
-                            Text(
-                                modifier = Modifier.testTag("FileSystemPopupFolderChoiceText"),
-                                text = folder.name,
-                                style = MaterialTheme.typography.bodyMedium,
-                                color = MaterialTheme.colorScheme.onSurface)
-                          }
-                    }
-                  } else {
-                    folderSubFolders.forEach { subFolder ->
-                      Box(
-                          modifier =
-                              Modifier.testTag("FileSystemPopupFolderChoiceBox" + subFolder.id)
-                                  .fillMaxWidth()
-                                  .background(
-                                      color = MaterialTheme.colorScheme.surface,
-                                      shape = RoundedCornerShape(8.dp))
-                                  .clickable {
-                                    folderViewModel.getSubFoldersOfNoStateUpdate(
-                                        subFolder.id,
-                                        onSuccess = { subFolders -> folderSubFolders = subFolders })
-                                    selectedFolder = subFolder
-                                  }
-                                  .padding(12.dp)) {
-                            Text(
-                                text = subFolder.name,
-                                style = MaterialTheme.typography.bodyMedium,
-                                color = MaterialTheme.colorScheme.onSurface)
-                          }
+                    verticalArrangement = Arrangement.spacedBy(4.dp)) {
+                    if (selectedFolder == null) {
+                        userRootFolders.value.forEach { subFolder(it)}
+
+                    } else {
+                        folderSubFolders.forEach {subFolder(it) }
                     }
-                  }
                 }
-            Box(
-                modifier = Modifier.fillMaxWidth().padding(16.dp),
-                contentAlignment = Alignment.Center) {
-                  Button(
-                      onClick = {
-                        onMoveHere(selectedFolder)
-                        onDismiss()
-                      },
-                      modifier = Modifier.testTag("MoveHereButton")) {
+                Box(
+                    modifier = Modifier.fillMaxWidth().padding(16.dp),
+                    contentAlignment = Alignment.Center) {
+                    Button(
+                        onClick = {
+                            onMoveHere(selectedFolder)
+                            onDismiss()
+                        },
+                        modifier = Modifier.testTag("MoveHereButton")) {
                         Text(text = stringResource(R.string.move_here))
```

Comment:
I would change the `R.string.move_here` string to just `Move` to simplify the button

Reply:
Alright I did it